### ISSUE TITLE
Add support for FSVD in permadisp

### ISSUE THREAD

**Issue by sfiligoi** [biocore/scikit-bio#1769]
Permadisp internally relies on pcoa.
Currently, there is no way to pass any parameters to pcoa, and it defaults to method="eigh".
Given how slow the eigh method is, being able to select FSVG instead would be great.

**Comment by sfiligoi** [comment]
In order to validate the usefulness of (the faster) FSVD vs eigh method, I computed permadisp with different pcoa methods on random subsamples going from 1k to 5k.
There is great correlation at all number of dimensions, although more dimensions is better.

Summary statistics over the 1k-5k range:
```
# FSVD/eigh F-value for identical input
        d=3     d=5     d=10    d=100
average 0.305   0.588   1.023   0.838
median  0.285   0.590   1.029   0.833
stdev   0.090   0.120   0.196   0.109
```

**Comment by sfiligoi** [comment]
<img width="1158" alt="Screen Shot 2021-06-03 at 11 43 59 AM" src="https://user-images.githubusercontent.com/2708665/120696116-059d5180-c461-11eb-8257-64a9efcab9f7.png">

**Comment by wasade** [comment]
Thanks, @sfiligoi! Adding a slightly different take on the raw results. And a brief note, these assessments were based off of randomly subsampling the published unweighted unifrac distance matrix from the EMP 150nt set, using empo_3 as the categorical variable.

![fstats](https://user-images.githubusercontent.com/474290/120697701-e7d0ec00-c462-11eb-89d1-11d483d51289.png)

**Comment by sfiligoi** [comment]
Some bechmark numbers comparing runtime of eight and fsvd.
(MacBook Pro, i7-8850H CPU)
```
       eigh   fsvd d=5   fsvd d=10   fsvd d=100
1000    39s         8s          8s          11s
2000   198s         9s         10s          15s
4000   940s        10s         10s          23s
```

**Comment by wasade** [comment]
Fixed in #1770
